components/GameCard/GameCard.js

```diff
@@ -0,0 +1,60 @@
+import styled from "styled-components";
+import { BsPersonCircle } from "react-icons/bs";
+
+export default function GameCard({ type, name, date, time, place, court }) {
+  return (
+    <section>
+      <StyledGameHeadline>{type}</StyledGameHeadline>
+      <StyledCardContainer>
+        <StyledCardPlayer>
+          <p>
+            <BsPersonCircle /> {name}
+          </p>
+        </StyledCardPlayer>
+        <StyledCardContent>
+          <article>
+            <h5>Date:</h5>
```

Comment:
You should not skip and headings on your page. h5 is the wrong element here and does not justify the page structure.